normandy/control/static/control/js/routes.js

```diff
@@ -3,13 +3,16 @@ import { Route } from 'react-router';
 import ControlApp from './components/ControlApp.jsx';
 import RecipeList from './components/RecipeList.jsx';
 import RecipeForm from './components/RecipeForm.jsx';
+import RecipePreview from './components/RecipePreview.jsx';
 import DeleteRecipe from './components/DeleteRecipe.jsx';
 
 export default (
   <Route component={ControlApp}>
     <Route path='control/'
       component={RecipeList}
-      ctaButton={{text: 'Add New', icon: 'plus', link: 'recipe/new/'}}
+      ctaButtons={[
```

Comment:
This is out-of-scope for this PR but I wonder if there's a better place we can specify these buttons than the Route components.